k8s/modules-ingress-mainnet.tf

```diff
@@ -1,6 +1,6 @@
 ############CID Checker#######################################
-module "ingress-kong_cid-checker-calibrationnet" {
-  count                            = local.id_mainnet_envs
+module "ingress-kong_cid-checker-mainnet" {
+  count                            = local.is_mainnet_envs
   source                           = "../modules/k8s_ingress"
```

Comment:
I regard you need the `is_kong_auth_header_enabled      = false` for the mainnet env as well